website/docs/02-separation-logic-and-biabduction.md

```diff
@@ -21,7 +21,7 @@ composing the reasoning chunks together.
 Separation logic is based on a logical connective <Math code="\\( * \\)" />
 called the _separating conjunction_ and pronounced "and separately". Separation
 logic formulae are interpreted over program allocated heaps. The logical formula
-<Math code="\\( A*B \\)" /> holds of a piece of program heap (a heaplet) when it
+<Math code="\\( A*B \\)" /> holds if a piece of program heap (a heaplet) when it
```

Comment:
I believe the original text was correct here and we can say "The formula holds *of* a piece of heap when ...". "if" is incorrect here.